ydb/docs/en/core/dev/troubleshooting/schemas/overloaded-shards.md

```diff
@@ -0,0 +1,23 @@
+# Overloaded shards
+
+## Description
+
+[Data shards](../../../concepts/glossary.md#data-shard) in {{ ydb-short-name }} might get overloaded for the following reasons:
+
+* A table is created without the [AUTO_PARTITIONING_BY_LOAD](../../../concepts/datamodel/table.md#AUTO_PARTITIONING_BY_LOAD) clause.
+
+    In this case {{ ydb-short-name }} does not split overloaded shards. If a shard has to process too many queries, it will use up all its CPU resources and, consequently, query response times will increase.
```

Comment:
It's not necessarily about CPU resources; it's about the fact that data shards are single-threaded and might not keep up with whatever is requested from them. Also, I'd describe this more carefully as there are a bit different cases: as far as I understand, `OVERLOADED` errors are returned more or less instantly when the load thresholds are exceeded (but you'll still lose time on retries), while the latency would be higher for queries that got accepted and wait for their turn to be executed (but they won't get the `OVERLOADED` error).